views/mempool.tmpl

```diff
@@ -61,36 +61,24 @@
                     <table class="table table-sm striped" id="mempool_votes">
                         <thead>
                             <th>Transactions ID</th>
-                            <th>Version</th>
-                            <th>Last Block Valid</th>
-                            <th>Issue ID</th>
-                            <th>Choice ID</th>
-                            <th class="text-right">Total DCR</th>
-                            <th>Size</th>
-                            <th class="text-right">Time in Mempool</th>
+                            <th>Vote Block</th>
+                            <th>Vote Index</th>
+                            <th>Vote Version</th>
+                            <th>Vote Valid</th>
```

Comment:
To me "Vote Valid" seems like it's somehow the validity of the vote itself.  I don't feel strongly about this, but perhaps "Vote Validation" would be clearer?

Reply:
Vote Choice